src/resourcemgr/tpl/folder.tpl

```diff
@@ -1,9 +1,11 @@
-<li>
-	<a 
-		data-path="{{path}}" 
-		data-display="{{relPath}}"
-		data-children-limit="{{childrenLimit}}"
-		href="#">
-		{{label}}
-	</a>
-</li>
+{{#if permissions.read }}
+	<li>
+		<a
+			data-path="{{path}}"
+			data-display="{{relPath}}"
+			data-children-limit="{{childrenLimit}}"
+			href="#">
```

Comment:
Same here, should be a button since it doesn't change URL

Reply:
I think is better to postpone taking a look at pressure to deliver this